projects/cli-files/go-ls/cmd/root.go

```diff
@@ -0,0 +1,30 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+)
+
+func Execute() {
+	arg := "."
```

Comment:
There's a function in os package that can help you get the current working directory. Can you think of why we should use that instead?

Reply:
Yes I found it `os.Getwd()` , I think we should use it because it gives an absolute path , making the code more reliable?

Reply:
That and also different operating systems might have a different understanding of what `"."` means. By using standard library, we can eliminate edge cases. You can take a look at the source code for `os.Getwd()` which will show you that there are quite a few if else statements :)